packages/patternfly-4/react-table/src/components/Table/Table.tsx

```diff
@@ -104,16 +104,16 @@ export type IAreActionsDisabled = (rowData: IRowData, extraData: IExtraData) =>
 // to be removed in future, this interface is no longer accurate
 export interface IDecorator extends React.HTMLProps<HTMLElement> {	
   isVisible: boolean;	
-  children?: React.ReactNode;	
+  children?: React.ReactNode;
 }
 
 export interface ICell {
   title?: string;
-  transforms?: ((...args: any) => any)[];
```

Comment:
I pushed a commit experimenting w/ this just now:
https://github.com/priley86/patternfly-react/commit/dd3b4443b0c64b3bfe3583a12f8adf9ab081bcc1

I'm seeing that something like this might work. I would just make sure that all use cases are covered in your integration tests though.

```
export interface ICell {
  title?: string;
  transforms?: ((label?: IFormatterValueType, rowData?: IRowData, columnIndex?: number, column?: IColumn, property?: string, rowIndex?: number, rowKey?: RowKeyType ) => { className: string; 'aria-sort': string; children: React.ReactNode; })[];
  cellTransforms?: ((label?: IFormatterValueType, rowData?: IRowData, columnIndex?: number, column?: IColumn, property?: string, rowIndex?: number, rowKey?: RowKeyType ) => { className: string; 'aria-sort': string; children: React.ReactNode; })[];
  columnTransforms?: ((label?: IFormatterValueType, rowData?: IRowData, columnIndex?: number, column?: IColumn, property?: string, rowIndex?: number, rowKey?: RowKeyType ) => { className: string; 'aria-sort': string; children: React.ReactNode; })[];
  formatters?: ((data?: IFormatterValueType, rowData?: IRowData, columnIndex?: number, column?: IColumn, property?: string, rowIndex?: number, rowKey?: RowKeyType ) => formatterValueType)[];
  cellFormatters?: ((data?: IFormatterValueType, rowData?: IRowData, columnIndex?: number, column?: IColumn, property?: string, rowIndex?: number, rowKey?: RowKeyType ) => formatterValueType)[];
```